src/Autofac.Extras.MvvmCross/AutofacMvxIocProvider.cs

```diff
@@ -482,7 +538,17 @@ public object Resolve(Type type)
             if (type == null)
                 throw new ArgumentNullException("type");
 
-            return _container.Resolve(type);
+            try
+            {
+                return _container.Resolve(type);
+            }
+            catch (DependencyResolutionException ex)
+            {
+                // throw MvxIoCResolveException
+                if (ex.InnerException is MvxIoCResolveException)
```

Comment:
Is this the recommended behaviour when integrating with a 3rd party container? OK to loose the extra exception information?